server/views/pages/order/monitoring-conditions/curfew-timetable.njk

```diff
@@ -11,9 +11,80 @@
 {% from "govuk/components/date-input/macro.njk" import govukDateInput %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "../../../partials/timeSpanInput.njk" import timeSpanInput %}
 
 {% block content %}
 
+{% macro _timetable(day, index, data, disabled) %}
+  <div class="timetable timetable-{{index}}">
+    {{ timeSpanInput({
+        id: "curfewTimetable-" + day + "-" + index + "-time",
+        namePrefix: "curfewTimetable[" + day + "][" + index + "][time",
+        nameSuffix: "]",
+        value: data.timeSpan.value,
+        errorMessage: data.timeSpan.error,
+        disabled: disabled
+      }) }}
+    
+    {{ govukCheckboxes({
+      idPrefix: "curfewTimetable-" + day + "-" + index + "-addresses",
+      name: "curfewTimetable[" + day + "][" + index + "][addresses][]",
+      values: data.addresses.values,
+      items: [
+        {
+          text: 'Primary address',
+          value: 'PRIMARY_ADDRESS',
+          disabled: disabled
+        },
+        {
+          text: 'Secondary address',
+          value: 'SECONDARY_ADDRESS',
+          disabled: disabled
+        },
+        {
```

Comment:
I do wonder if we should have an option for Third address if we can't send this to Serco